.github/workflows/publish.yml

```diff
@@ -0,0 +1,117 @@
+name: publish
+
+on:
```

Comment:
To confirm: no PRs can trigger this workflow, right?

Reply:
The environment variables needed for publishing are also locked behind the production environment which is only accessible from main